packages/nft/src/NFTHandler.ts

```diff
@@ -117,4 +120,70 @@ export class NFTHandler {
       tokenIds.map(tokenId => this.get(nftManager, storage, tokenId, nftFormat))
     )
   }
+
+  /**
+   * It takes a tokenId, and returns a transaction object
+   * @param {NFT721Manager} nftManager - NFT721Manager - The NFT721Manager instance that will be used
+   * to set the NFT tokenURI.
+   * @param {StorageType} storage - StorageType - The storage to save the NFT metadata.
+   * @param {NFT_METADATA_FORMAT} nftFormat - The format of the NFT metadata.
+   * @param {string} tokenId - The tokenId of the NFT you want to set.
+   * @param {NFTMetadata} nftMetadata - The metadata you want to set.
+   * @param {string} image - Base64 of the image you want to set.
+   * @returns The transaction object
+   */
+  static async change(
+    nftManager: NFT721Manager,
+    storage: StorageType,
+    nftFormat: NFT_METADATA_FORMAT,
+    nftData: { tokenId: string; image?: string; nftMetadata?: NFTMetadata }
+  ): Promise<interfaces.TransactionResponseExtended | undefined> {
+    const { tokenId, image, nftMetadata } = nftData
+
+    if (!nftManager.isActive) {
+      throw MUST_ACTIVATE
+    }
+
+    if (!nftManager.implements('setTokenURI', ['uint256', 'string'])) {
```

Comment:
We should have done this before, but it may be best to make a decorator out of this, by adding [this plugin](https://babeljs.io/docs/en/babel-plugin-proposal-decorators), to improve code reusability and cleanliness. Since is basically always the same. Same goes for isActive.
If possible.

We can do it in a different PR, and update all the places that are doing this at once.